scripts/changes-include-dir

```diff
@@ -12,8 +12,7 @@ set -euo pipefail
 DIRS=$@
 
 for DIR in $DIRS; do
-    git diff origin/master...$CIRCLE_BRANCH --name-status | grep $DIR
-    if [ $? -eq 0 ]; then
+    if [ $(git diff signalfx/master...$CIRCLE_BRANCH --name-status | grep -c $DIR) -gt 0 ]; then
```

Comment:
Can't this just be `if git diff signalfx/master...$CIRCLE_BRANCH --name-status | grep -c $DIR; then`?

Reply:
That doesn't swallow the stdout and leaves some litter:

```
CIRCLE_BRANCH=update_ansible_test scripts/changes-include-dir deployments asdf asdfasga asdgasdf cgggg adsfasdf changes
0
0
0
0
0
0
1
```